HappyAnding/HappyAnding/Views/SearchView.swift

```diff
@@ -101,7 +101,7 @@ struct SearchView: View {
     
     var proposeView: some View {
         VStack(alignment: .center) {
-            Text("\(searchText)의 결과가 없어요.\n원하는 단축어를 제안해보는건 어떠세요?").multilineTextAlignment(.center)
+            Text("\"\(searchText)\"의 결과가 없어요.\n원하는 단축어를 제안해보는건 어떠세요?").multilineTextAlignment(.center)
```

Comment:
디자인상에서는 큰따옴표가 아닌 작은따옴표로 알고있는데 혹시 큰따옴표로 하신 이유가 있으신가요?
```suggestion
            Text("\'\(searchText)\'의 결과가 없어요.\n원하는 단축어를 제안해보는건 어떠세요?").multilineTextAlignment(.center)
```